astro/src/content/docs/get-started/core-concepts/integration-points.mdx

```diff
@@ -69,30 +70,36 @@ Each JWT has a header, a payload and a signature. Here's a diagram of a JWT:
 
-| Feature                                                                                                 | Cookie                                           | Server Side Session            |
-|---------------------------------------------------------------------------------------------------------|--------------------------------------------------|--------------------------------|
-| Allows for easy horizontal scaling                                                                      | Yes                                              | No                             |
-| Revocation possible                                                                                     | [Yes](/learn/expert-advice/tokens/revoking-jwts) | Yes                            |
-| Revocation easy                                                                                         | No                                               | Yes                            |
-| Sticky sessions or session datastore required                                                           | No                                               | Yes                            |
-| Token transparently sent on requests (to APIs, etc)                                                     | Yes                                              | N/A                            |
-| Token can be presented to APIs on domains other than that in which the cookie setting component resides | No                                               | Yes (via server side requests) |
+| Feature                                                                                                 | Cookie                                           | BFF/Server-side Session        | Native Secure Storage          |
+|---------------------------------------------------------------------------------------------------------|--------------------------------------------------|--------------------------------|--------------------------------|
+| Allows for easy horizontal scaling                                                                      | Yes                                              | No                             | Yes |
+| Revocation possible                                                                                     | [Yes](/learn/expert-advice/tokens/revoking-jwts) | Yes                            | Yes |
+| Revocation easy                                                                                         | No                                               | Yes                            | No  |
+| Sticky sessions or session datastore required                                                           | No                                               | Yes                            | No  |
+| Token transparently sent on requests (to APIs, etc)                                                     | Yes                                              | N/A                            | No  |
```

Comment:
```suggestion
| Token transparently sent on requests (to APIs, etc) -- easy to "see" and debug                                                     | Yes                                              | N/A                            | No  |
```

Reply:
"Transparent" is a vague word (to me anyway) - my edit is not perfect here, but thinking about ways to communicate this idea of playing sing-a-long with the token is "harder" using some methods.

Reply:
reworked.